example-workflows/other_WDL/README.md

```diff
@@ -0,0 +1,179 @@
+# AWS HealthOmics Protein Workflows
```

Comment:
This README needs to update references to protein folding to ones indicative of the workflows in the grouping - i.e. HLA

Reply:
yes this change was pushed last night.